kedro-datasets/kedro_datasets/snowflake/snowpark_dataset.py

```diff
@@ -10,7 +10,9 @@
 logger = logging.getLogger(__name__)
 
 
-class SnowparkTableDataSet(AbstractDataSet):
+class SnowparkTableDataSet(
+    AbstractDataSet
+):  # pylint:disable=too-many-instance-attributes
```

Comment:
Maybe we should just ignore this on the repo level, it's not really a relevant check for the datasets.